azurerm/internal/services/web/tests/resource_arm_function_app_test.go

```diff
@@ -203,17 +202,17 @@ func TestAccAzureRMFunctionApp_connectionStrings(t *testing.T) {
 				Config: testAccAzureRMFunctionApp_connectionStrings(data),
 				Check: resource.ComposeTestCheckFunc(
 					testCheckAzureRMFunctionAppExists(data.ResourceName),
-					resource.TestCheckResourceAttr(data.ResourceName, "connection_string.0.name", "Example"),
-					resource.TestCheckResourceAttr(data.ResourceName, "connection_string.0.value", "some-postgresql-connection-string"),
-					resource.TestCheckResourceAttr(data.ResourceName, "connection_string.0.type", "PostgreSQL"),
+					resource.TestCheckResourceAttr(data.ResourceName, "connection_string.163594034.name", "Example"),
+					resource.TestCheckResourceAttr(data.ResourceName, "connection_string.163594034.value", "some-postgresql-connection-string"),
+					resource.TestCheckResourceAttr(data.ResourceName, "connection_string.163594034.type", "PostgreSQL"),
```

Comment:
It just comes to my mind that would this check path accept built-in functions in hcl? such as a `toList` function or something. is there such a thing?

Reply:
```tolist``` not working in this test case